src/main/scala/chisel3/tester/TreadleBackend.scala

```diff
@@ -17,9 +17,12 @@ class TreadleBackend[T <: MultiIOModule](dut: T,
     val dataNames: Map[Data, String], val combinationalPaths: Map[Data, Set[Data]],
     tester: TreadleTester)
     extends BackendInstance[T] with ThreadedBackend {
+
+  type CycleTracker = mutable.Map[Clock, Int]
```

Comment:
returned to original form. I had changed this because it was the only obstacle remaining for this file to be style check clean. I try to keep my files that way as it helps IntelliJ catch stuff for me, but this is your code. I had introduced the type declaration trying to minimize the boilerplate to get to clean, I wasn't entirely satisfied by it.